shell/ci/release/confluence-publish.sh

```diff
@@ -75,4 +75,5 @@ while read -r file; do
   else
     info_sub "no space directive found, skipping ${file}"
   fi
-done < <(git diff --name-only HEAD HEAD~1 -- '*.md')
+  readarray -t releaseCommits < <(git log --pretty=format:'%H' -n 2 --grep='^chore' --invert-grep)
+done < <(git diff --name-only "${releaseCommits[@]}" -- '*.md')
```

Comment:
We _might_ want to consider just doing commits since last tag, but this is fine with me for now

Reply:
That's a great idea! I can grab the first non-chore commit to represent the current commit and compare it agains the last tag commit.

I also just realized that `git diff` can accept a commit range. Let me get this updated.

Reply:
@jaredallard let me know what you think of the new approach.